framework/src/modules/network/index.js

```diff
@@ -0,0 +1,66 @@
+const { config } = require('./defaults');
+const Network = require('./network');
+const BaseModule = require('../base_module');
+
+/* eslint-disable class-methods-use-this */
+
+/**
+ * Network module specification
+ *
+ * @namespace Framework.Modules
+ * @type {module.NetworkModule}
+ */
+module.exports = class NetworkModule extends BaseModule {
+	/* eslint-disable-next-line no-useless-constructor */
+	constructor(options) {
+		super(options);
+	}
+
+	static get alias() {
+		return 'network';
+	}
+
+	static get info() {
+		return {
+			author: 'LiskHQ',
+			version: '0.1.0',
+			name: 'lisk-core-network',
```

Comment:
May be we should update this naming convention for all modules, for reference this name will reflect the npm package name when we load module directly from a custom npm package, or when we publish these modules as independent nam packages.

Reply:
Created separate issue #3467